app/services/category_hashtag_data_source.rb

```diff
@@ -9,11 +9,13 @@ def self.icon
   end
 
   def self.category_to_hashtag_item(guardian_categories, category)
-    category = Category.new(category.slice(:id, :slug, :name, :parent_category_id))
+    category =
+      Category.new(category.slice(:id, :slug, :name, :parent_category_id, :description))
```

Comment:
Do we need to create new object? Why not access category attributes directly?

Reply:
Because it's a hash, it's easier to convert to an actual Category since then we can access methods like this on the model:

https://github.com/discourse/discourse/blob/952dadb062233556ecc6be7aa6c12701b2f600c1/app/models/category.rb#L769-L773